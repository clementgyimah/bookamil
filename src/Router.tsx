import React from 'react'
import { HomePage } from './pages/home/HomePage'
import { ErrorBoundary } from './components'

const Router = () => {
  return (
    <ErrorBoundary props={undefined}>
      <HomePage />
    </ErrorBoundary>
  )
}

export { Router }
