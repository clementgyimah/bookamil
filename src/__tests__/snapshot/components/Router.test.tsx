import renderer from 'react-test-renderer'
import { Router } from '../../../Router'

test('home page snapshot test', () => {
  const routerSnap = renderer.create(<Router />)
  const routerSnapTree = routerSnap.toJSON()
  expect(routerSnapTree).toMatchSnapshot()
})
