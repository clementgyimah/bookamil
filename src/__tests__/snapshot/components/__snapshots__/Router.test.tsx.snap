// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home page snapshot test 1`] = `
<div
  className="main-container"
>
  <header>
    <div
      className="app-details-container"
    >
      <span
        className="title40"
        id="app-name-text"
      >
        Bookamil
      </span>
      <span
        className="title20"
        id="short-description-text"
      >
        Browse millions of books. Read your favourite books here
      </span>
    </div>
  </header>
  <div
    className="home-body"
  >
    <div
      className="category-title-container"
    >
      <span
        className="normal-text25"
      >
        Most popular
      </span>
    </div>
    <div
      className="category-content"
    >
      <div
        className="spinner-main-container"
      >
        <div
          aria-label="book-loader"
          className="spinner-style spinner-animation"
        >
          <span />
        </div>
      </div>
    </div>
  </div>
</div>
`;
